### ISSUE TITLE
Duplicate sum declarations with inheritance

### ISSUE THREAD

**Issue by hajduakos** [SRI-CSL/solidity#53]
The following example will result in two conflicting declarations for the sum variable
```solidity
/// @notice invariant __verifier_sum_int(map) == total
contract Base {
    mapping(address=>int) map;
    int total;
}

/// @notice invariant __verifier_sum_int(map) == total
contract Derived is Base {
}
```

**Comment by hajduakos** [comment]
Fixed in 162ec9e, should close the issue when the branch is merged

**Comment by hajduakos** [comment]
Merged, closing issue
